setup.py

```diff
@@ -43,7 +43,7 @@ def main():
         author='holger krekel',
         author_email='holger@merlinux.eu',
         packages=['tox'],
-        entry_points={'console_scripts': ['tox=tox.session:run_main',
+        entry_points={'console_scripts': ['tox=tox:cmdline',
```

Comment:
I don't understand this rename; the cmdline actually shows up as not a valid dictionary entry, hence why I renamed it to run_main instead, to be more explicit

Reply:
e.g. in the English dictionary the word cmdline does not exist

I would consider this bad naming in the past, having the proxy names help correct it internally imho, although if you really insist we can keep it as such